course/meetings/mie-matthew-mackay-2017-03-13.md

```diff
@@ -0,0 +1,50 @@
+
+---
+contact: Matthew Mackay
+department: Mechanical and Industrial Engineering
+uoftcoders: Madeleine, Joel
+date: 2017-03-13
+---
+
+# Summary
+- His ideal course would be somewhat engineering specific and attractive to both mechanical and industrial engineers. He seemed more interesting in uniting these two engineering streams than bridging gaps to other departments.
+    - If there was a course that bridged the interests of the Mechanical and Industrial streams, that would be very attractive to the department.
+- Want more programming in general for his undergrads.
+    - Currently a big gap between taking a few courses in first year and then basically nothing until fourth year.
+        - At least for Mechanical engineers where programming is a hard sell, the industrial stream has decent programming experience.
+    - Currently mostly dependent on student electives.
+- The RQM could potentially replace and update their numerical methods course.
+- RQM course would be closer to industrial engineering where they do more research.
+    - There is also financial engineering that could be interesting
+    - Environmental engineering could be a match with EEB?
+- Grad students are allowed to teach, but they mostly get professional engineers.
+- Will put us in contact with some of their teaching grad students for tips.
+- There might be some resistance from higher levels that tend to like the status quo.
+
```

Comment:
Suggested to contact [Bruno Korst](https://www.ece.utoronto.ca/undergraduates/hardware-computer-labs/communications-lab/) in ECE - he's a lab manager with clout, might be interested in the course.